scipy/linalg/matfuncs.py

```diff
@@ -668,3 +669,48 @@ def rounded_sign(x):
         return S0
     else:
         return S0, errest
+
+
+def khatri_rao(a, b):
+    """
+    Khatri-rao product
+
+    Parameters
+    ----------
+    a:  (I, K) ndarray
+        Input array
+    b:  (J, K) ndarray
+        Input array
+
+    Returns
+    -------
+    C:  ((IJ), K) ndarray
+        Khatri-rao product of 'a' and 'b'.
```

Comment:
quotes needs to be double backticks to render it as variable in the html docs

Reply:
I will fix it